config/locales/gobierto_visualizations/views/ca.yml

```diff
@@ -24,6 +24,19 @@ ca:
         contractor: Contractor
         contracts: Contractes
         contracts_assigned_to: Contractes adjudicats a
+        contracts_show:
+          assignees: Adjudicataris
+          awarding: Adjudicació
+          awarding_entity: Entitat adjudicadora
+          batch: Lot
+          bid_description: Pressupost base de licitació sense impostos
+          bidders: Licitadors
+          contract_amount: Import d'adjudicació
+          entity: Entitat
+          process: Procés
+          question_description: "¿Que % de l'pressupost suposa aquest contracte?"
```

Comment:
This error comes from GoogleTranslate. Article contraction is wrong.
Use https://www.softcatala.org/traductor/

Reply:
Què % del pressupost suposa aquest contracte_

Reply:
Done!